README.md

```diff
@@ -12,8 +12,10 @@ with:
 - Digital signatures
 - Cryptographic hashing
 
-## Blockchain main goal
-Watch [this video](https://www.youtube.com/watch?v=RplnSVTzvnU) (Understanding Blockchain function). (Est. time: 15 minutes)
+## Blockchain
+
+1. Read [What is Blockchain?](https://medium.com/coinmonks/blockchain-technology-explained-7d68c4435aca). (Est.time: 10 minutes)
+2. Watch [this video](https://www.youtube.com/watch?v=_160oMzblY8). (Est. time: 18 minutes)
```

Comment:
Nice. Personal opinion (non-blocker) why don't we add the video title/name to this link?

Reply:
Perfect. I agree.